st3/lsp_utils/server_npm_resource.py

```diff
@@ -85,7 +86,11 @@ def setup(self) -> None:
             return
 
         self._initialized = True
-        self._package_cache_path = os.path.join(sublime.cache_path(), self._package_name)
+        if self._install_in_cache:
+            data_dir = sublime.cache_path()
+        else:
+            data_dir = os.path.normpath(os.path.join(sublime.cache_path(), '..'))
+        self._package_cache_path = os.path.join(data_dir, 'LSP-servers', self._package_name)
```

Comment:
Can we call it LSP-Data instead of LSP-servers? It could also be used for log files and such. All other directories in the $DATA dir are also titlecase.